SyntheticGeometry/SQC/Consequences.lagda.md

```diff
@@ -88,6 +88,7 @@ generalized-field-property : {n : _} → (xs : FinVec ⟨ k ⟩ n) → ¬(xs ≡
 generalized-field-property xs xs≢0 =
   Consequences.onFGIdeals k k-local xs 1∈⟨xs⟩
   where
+    open AlgebraHoms
```

Comment:
There's already an `open` statement for the composition operation in [line 115](https://github.com/felixwellen/synthetic-geometry/pull/28/files#diff-b29abd38c657120a2f664991bb95663724ff1110d9e3e4342129debfd04a2766L115). The pull request agda/cubical#1038 replaced the `_∘a_` syntax declaration for `compAlgebraHom` by a corresponding operator declaration so that the `open` statement in that line should read `open AlgebraHoms using (_∘a_)` now instead of `open AlgebraHoms using (compAlgebraHom)`.

Reply:
Thanks! Now done in another PR